src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System.Private.Runtime.InteropServices.JavaScript.csproj

```diff
@@ -32,6 +59,12 @@
     <Compile Include="System\Runtime\InteropServices\JavaScript\Function.cs" />
     <Compile Include="System\Runtime\InteropServices\JavaScript\JSObject.References.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetsBrowser)' != 'true'">
+    <None Include="System\Runtime\InteropServices\JavaScript\Public\JavaScriptMarshal.PlatformNotSupported.cs" />
+  </ItemGroup>
```

Comment:
This project only targets Browser only so this condition will never be true. Also the `$(TargetsBrowser)` property doesn't exist anymore (the replacement now is `'$(TargetPlatformIdentifier)' == 'Browser'`: https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/project-guidelines.md#targetframework-conditions (Example 3).

```suggestion
```

Reply:
Thanks Viktor!